_1327/user_management/migrations/0003_userprofile_language.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-07-23 17:24
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+		('user_management', '0001_initial'),
+        # ('user_management', '0002_auto_20170306_1738'),
```

Comment:
what's this?

Reply:
Reordering the migrations of the user_management so it can handle the newly added fields